cfme/v2v/migrations.py

```diff
@@ -0,0 +1,329 @@
+                            }
+                        ]
+                    }
+                }
+            }
+    """
+    title = Text(locator='.//h4[contains(@class,"modal-title")]')
+    general_view = View.nested(InfraMappingWizardGeneralView)
+    clusters_view = View.nested(InfraMappingWizardClustersView)
+    datastores_view = View.nested(InfraMappingWizardDatastoresView)
+    networks_view = View.nested(InfraMappingWizardNetworksView)
+    results_view = View.nested(InfraMappingWizardResultsView)
+
+    def after_fill(self, was_change):
+        if was_change:
+            self.results_view.next_btn.click()
+
+
+class MigrationDashboardView(BaseLoggedInPage):
+    create_infrastructure_mapping = Text(locator='(//a|//button)'
+        '[text()="Create Infrastructure Mapping"]')
+    create_migration_plan = Text(locator='(//a|//button)[text()="Create Migration Plan"]')
+
+    @property
+    def is_displayed(self):
+        return self.navigation.currently_selected == ['Compute', 'Migration']
+
+
+class AddInfrastructureMappingView(View):
+    wizard_form = InfraMappingWizard()
+
+    @property
+    def is_displayed(self):
+        return self.wizard_form.title.text == 'Infrastructure Mapping Wizard'
+
+
+class AddMigrationPlanView(View):
+    title = Text(locator='.//h4[contains(@class,"modal-title")]')
+    name = TextInput(name='name')
+    description = TextInput(name='description')
+    back_btn = Button('Back')
+    # Since next is a keyword, suffixing it with btn and other two
+    # because want to keep it consistent
+    next_btn = Button('Next')
+    cancel_btn = Button('Cancel')
+
+    @property
+    def is_displayed(self):
+        return self.title.text == 'Migration Plan Wizard'
+
+
+# Collections Entities
+
+
+@attr.s
+class InfrastructureMapping(BaseEntity):
+    """Class representing v2v infrastructure mappings"""
+    name = attr.ib()
+    description = attr.ib(default=None)
+    form_data = attr.ib(default=None)
+
+
+@attr.s
+class InfrastructureMappingCollection(BaseCollection):
+    """Collection object for Migration mapping object"""
+    ENTITY = InfrastructureMapping
+
+    def create(self, form_data):
+        infra_map = self.instantiate(
+            name=form_data['general_view']['name'],
+            description=form_data['general_view']['description'],
+            form_data=form_data
+        )
+        add_infrastucture_mapping_view = navigate_to(self, 'Add')
+        add_infrastucture_mapping_view.wizard_form.fill(form_data)
+        return infra_map
+
+# TODO: Next Entity and Collection classes are to be filled by Yadnyawalk(ytale),
+# which he will submit PR for once my PR merged.
+
+
+@attr.s
+class MigrationPlan(BaseEntity):
+    """Class representing v2v Migration Plan"""
+    category = 'migrationplan'
+    string_name = 'Migration Plan'
+
+
+@attr.s
+class MigrationPlanCollection(BaseCollection):
+    """Collection object for Migration Plan object"""
+    ENTITY = MigrationPlan
+
```

Comment:
No create method to use the 'Add' nav destination and view that was written?

Reply:
added # TODO as we discussed on irc